gctf/protocols/protocol_gctf.py

```diff
@@ -299,7 +347,13 @@ def _restimateCTF(self, ctfId):
         mic = ctfModel.getMicrograph()
         micFn = mic.getFileName()
         micDir = self._getMicrographDir(mic)
-        micFnCtf = self._getTmpPath(pwutils.replaceBaseExt(micFn, 'ctf'))
+
```

Comment:
This bit seems repeated, maybe worth a small function for it?

Reply:
this will be changed anyway when batch func will be added